app/scripts/views/force_auth.js

```diff
@@ -32,17 +37,35 @@ function (p, BaseView, FormView, SignInView, Template, Session) {
       this.user.clearSignedInAccount();
     },
 
-    context: function () {
-      var fatalError = '';
-      var email = this.relier.get('email');
+    beforeRender: function () {
+      var self = this;
+      var relier = self.relier;
+      var email = getEmail(relier);
+
       if (! email) {
-        fatalError = t('/force_auth requires an email');
+        self._fatalError = t('/force_auth requires an email');
+        return;
```

Comment:
Good idea. We still want to render a screen, it should just show the fatal error message. 

We have a couple of different ways to handle screen startup errors, it would be great to come up with something generic and settle on that. Maybe something like return an `Error` from `beforeRender` and instead of rendering the screen's template, we render a generic error template with the fatal error. Or, maybe we use the functionality in https://github.com/mozilla/fxa-content-server/pull/1982 to redirect to `400.html` with the error. So many options!